src/value/element_stream_reader.rs

```diff
@@ -416,17 +381,12 @@ impl IonReader for ElementStreamReader {
     }
 
     fn ion_version(&self) -> (u8, u8) {
-        todo!()
+        // An `Element` doesn't have an Ion version associated with it
+        // Since `Element`s are an in-memory representation fo Ion data, all versions of 1.x share the same Ion  version.
```

Comment:
nit: extra whitespace
```suggestion
        // Since `Element`s are an in-memory representation fo Ion data, all versions of 1.x share the same Ion version.
```